src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
-
+import core.machines.*;
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine excavator = new Excavator();
+        Machine bulldozer = new Bulldozer();
+        Machine truck = new Truck();
+        Machine[] machines = new Machine[] {excavator, bulldozer, truck};
+        for(Machine machine : machines) {
```

Comment:
There should not be an empty space between the 'for' keyword and the opening parenthesis to adhere to Java coding standards.